### ISSUE TITLE
Add codes in languages other than the ones in the repository

### ISSUE THREAD

**Issue by pratyushmp** [pratyushmp/Hacktoberfest_2k-19#10]

**Comment by RmViana** [comment]
Can I add some Javascript content?

**Comment by pratyushmp** [comment]
Yes you can. Just make a new folder named Java Script and commit

**Comment by RmViana** [comment]
Ok, thanks!

**Comment by RmViana** [comment]
I'm trying to commit my changes, but I think there's a problem with Python's folder. Somehow, after trying for at least three times to repeat the process of forking the repository and cloning it to my local computer (and even executing `git pull` right after it), I end up with an empty Python folder:

![python_folder](https://user-images.githubusercontent.com/19740175/67160013-4e52d600-f322-11e9-9f36-83bb5f3ca3e1.JPG)

Should I create a new branch and submit my PR there?

**Comment by pratyushmp** [comment]
Ok. That's weird. Did you try manually uploading your code to github? If you fail even then, create a new branch named "python" and make a pull request!

**Comment by RmViana** [comment]
I've uploaded the code manually, and now looks like it worked. Thank you!

**Comment by prachijustin** [comment]
I had the same issue. The issue is caused by the folder having name **# Run this script as root     import time  from datetime import datetime as dt     # change hosts path according to your OS  hosts_path = _**. I request the owner to edit this name since it containes one Py file as well.

Thanks.

**Comment by pratyushmp** [comment]
@prachijustin  Would be very helpful if you could do it! Thanks!

**Comment by harshalkh** [comment]
I have added new folder for GO with bubble sort algorithm. Review PR #75
